### PR TITLE
i1 class methods

### PR DESCRIPTION
**Pull request by ryandbarnett**
Adds a create_games, create_teams, and create_game_teams class methods to their corresponding classes.
Updates stat tracker to use these class methods instead of our collection classes.
Deletes unused mock variables in game statistics test.

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -35,4 +34,13 @@ def initialize(game_team_data)
     @giveaways = game_team_data[:giveaways].to_i
     @takeaways = game_team_data[:takeaways].to_i
   end
+
+  def self.create_game_teams(path)
```
Comment (dhalverson) [inline lib/game_team.rb:38]: This looks good!

**Review by dhalverson — APPROVED** [review]
Approving this, everything looks good

**Comment by hashmaster3k** [comment]
Shall we delete the non-needed collections files?

**Review by hashmaster3k — APPROVED** [review]

**Comment by ryandbarnett** [comment]
Yes Hash good catch. I deleted them. All the tests still function/pass.

**Pull request merged**